src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.AgentPool.ps1

```diff
@@ -450,4 +454,60 @@ class AgentPool: ADOSVTBase
 
         return $controlResult;
     }
+
+    hidden [ControlResult] CheckBroaderGroupAccessAutomatedFix ([ControlResult] $controlResult) {
+        try {
```

Comment:
Although command gives "Fixed" status, the new permissions are not getting reflected
pls check

Reply:
Fixed